ui/packages/catalog-ui-search/src/main/webapp/react-component/export-results/index.js

```diff
@@ -8,20 +8,29 @@ const CustomElements = require('js/CustomElements')
 const Component = CustomElements.registerReact('export-results')
 
 const capitalize = value => value.charAt(0).toUpperCase() + value.slice(1)
+const haveSearchResults = state => state && state.activeSearchResults && state.activeSearchResults.length > 0
 
 class ExportResults extends React.Component {
   constructor(props) {
     super(props)
 
     const passedProps = (props && props.options && props.options.props) || {}
+    const activeSearchResults = {activeSearchResults: props.state && props.state.activeSearchResults || []}
     const defaultExportSize = Object.keys(passedProps.export)[0]
     this.state = {
       ...passedProps,
+      ...activeSearchResults,
       exportSizeValue: defaultExportSize,
       exportFormats: [],
     }
   }
 
+  componentDidUpdate = async (previousProps) => {    
+    if (previousProps.state.activeSearchResults !== this.props.state.activeSearchResults) {
```

Comment:
❓ @andrewkfiedler Can we compare an array of objects like this? Here's an example where javascript doesn't act as I expected with a similar type of comparison

```
[1, 2, 3] === [1, 2, 3]
> false
```

Reply:
In this instance yes, because we're comparing references. `previousProps` is the state of `this.props` prior to this update. If the update introduces no changes to `this.props.state.activeSearchResults` no changes took place.

Reply:
This is equivalent to
```
const list = [1, 2, 3]
list === list

list !== [...list, 4]
```